cmd/rosa/main.go

```diff
@@ -94,3 +103,18 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func GetLatestROSAVersion() string {
+	client := github.NewClient(nil)
```

Comment:
What would happen if a user doesn't have access to GH for some reason (e.g. FW rules, GH outage)?